README.md

```diff
@@ -95,7 +74,7 @@ Public CNCF TOC channel: Join the [CNCF slack](https://slack.cncf.io/) and we ar
 
 This is our [voting policy](docs/voting.md). Only TOC members can cast binding votes. (+1 Binding)
 
-## Technical Advisory Groups
+## Technical Advisory Groups being phased out
```

Comment:
very small nit, but we seem to use Title Case most places for headings in this document so this looks slightly off when reading through the rest of the doc?